modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js

```diff
@@ -92,15 +107,28 @@ export default withRouter(
 			}
 		}, [question, page, pageSize]);
 
+		const getErrorObject = useCallback((error) => {
```

Comment:
Do you need the function in the component? You can extract it to an util class and avoid the dependency/useCallback IMHO.